src/playground.js

```diff
@@ -0,0 +1,3 @@
+let now = new Date(Date.now());
+
+console.log(now.getHours());
```

Comment:
Does this file need to be checked in?

Reply:
it does not. Do I just add it to the git ignore?

Reply:
Yea I think you would just say `src/playground.js`. However I don't remember how to remove it from git. Maybe since it isn't in the repo yet you could just delete it (or move it out the project), commit the deletion, then add it back into the project.